sdk/cosmos/azure-cosmos/azure/cosmos/_synchronized_request.py

```diff
@@ -136,14 +136,17 @@ def _Request(global_endpoint_manager, request_params, connection_policy, pipelin
             connection_verify=kwargs.pop("connection_verify", is_ssl_enabled),
             **kwargs
         )
-
+    #Will make updating diagnostics less verbose
+    ir = response.http_response.internal_response
```

Comment:
SDKs should not interact with `internal_response`.
There are [already policies in the SDK pipeline](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/cosmos/azure-cosmos/azure/cosmos/_cosmos_client_connection.py#L192-L195) for diagnostics, and any additions that need to be added should build on those existing policies if needed.